LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2019, OpenFAM
+Copyright (c) 2019, Hewlett Packard Enterprise Development, LP.
```

Comment:
Can you also add the link to BSD clause in end( See https://spdx.org/licenses/BSD-3-Clause). Similar to other files

Reply:
Updated the LICENSE file with the above line similar to License text in other files.

Regards,
Faizan